pkg/operations/kubernetesupgrade/upgrader.go

```diff
@@ -255,6 +256,17 @@ func (ku *Upgrader) upgradeMasterNodes(ctx context.Context) error {
 	}
 
 	upgradedMastersIndex := make(map[int]bool)
+
+	for _, vm := range *ku.ClusterTopology.MasterVMs {
```

Comment:
should this loop over `UpgradedMasterVMs` instead?

Reply:
That array will be empty at that point as none of the masters have been upgraded